quetz/tests/test_jobs.py

```diff
@@ -598,16 +598,20 @@ def test_post_new_job_manifest_validation(
 
 
 @pytest.mark.parametrize("user_role", ["owner"])
-def test_post_new_job_invalid_items_spec(auth_client, user, db, dummy_job_plugin):
+def test_post_new_job_invalid_items_spec(
+    auth_client, user, db, dummy_job_plugin, mocker
+):
     # items_spec=None is not allowed for jobs
     # (but it works with actions)
     manifest = "quetz-dummyplugin:dummy_func"
+    dummy_func = mocker.Mock()
```

Comment:
Why did you have to add this?

Reply:
Without this, the error raised is:

```
>       assert "Input should be a valid string" in msg[0]['msg']
E       assert 'Input should be a valid string' in "Value error, invalid function 'dummy_func' name in plugin 'quetz-dummyplugin'"
```

I guess the validation order changed in pydantic. The "manifest" value is checked before "items_spec".